files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/handlers/main.yaml

```diff
@@ -23,3 +23,12 @@
 - command: /usr/bin/google_instance_setup
   listen: "gcp config changed"
   when: not ansible_is_chroot
+
+#
+# Restart the NFS server to pickup config changes
+#
+- name: restart nfs-server
```

Comment:
we use "listen" above.. should we standardize on "name" or "listen".. I'd have to look at the docs to know how the two differ..